modules/howtos/pages/distributed-acid-transactions-migration-guide.adoc

```diff
@@ -0,0 +1,185 @@
+= Distributed Transactions Migration Guide
```

Comment:
I think this should probably move to the `Project Docs` section, under Compatibility.
https://docs-staging.couchbase.com/java-sdk/current/project-docs/compatibility.html 

But, I'm happy to keep this here for now in the interest of getting this on staging.
We can re-arrange where this goes in a separate PR.

Reply:
Good idea, and sounds like a plan.